src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -185,6 +185,10 @@ public enum ErrorCode {
 
   // API Table Error Codes
   API_TABLE_FEATURE_NOT_ENABLED("API Table feature is not enabled"),
```

Comment:
Just realized that this should probably not have `API_` prefix -- so if you change new ones to start with `TABLE_`, could change this one as well.